doc/source/operations.rst

```diff
@@ -55,23 +55,60 @@ Once the appropriate profile has been selected, it can be applied by running the
+
+.. code-block:: yaml
+
+   suse_osh_image_version: "ocata"
+
+It is also possible to update an individual image or subset of images to a different version, rather than making a global change. To do this, it will be necessary to manually edit the versions.yaml file located in socok8s/site/soc/software/config/. Locate the images that need to be changed in the "images" section of the file and modify the line to include the desired version. For example, to use the "rocky" version for the heat_api image, change the following line in versions.yaml from
+
+.. code-block:: yaml
+
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+
+to
+
+.. code-block:: yaml
 
-Update OpenStack Services
--------------------------
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:rocky"
 
-Update secrets, passwords and certificates
+Updating Individual Images and Helm Charts
 ------------------------------------------
 
-Update this repository
-----------------------
+The versions.yaml file can also be used for more advanced update configurations such as using a specific image or Helm chart source version. 
 
-The SUSE Containerized OpenStack repository can be updated by performing the git pull command from the socok8s directory:
+.. note::
+   
+   The following configuration options are not supported for the SUSE Containerized OpenStack Technical Preview. Please proceed at your own risk.
 
-.. code-block:: console
+To specify the use of an updated or customized image, locate the appropriate image name in socok8s/site/soc/software/config/versions.yaml and modify the line to include the desired image location and tag. For example, to use a new heat_api image, modify its entry with the new image location:
+
+.. code-block:: yaml
+
+   heat_api: "registry_location/image_directory/image_name:tag"
+
+Similarly, the versions.yaml file can be used to retrieve a specific version of any Helm chart being deployed. To do so, it will be necessary to provide a repository location, type, and a reference. For example, to use a specific version of the Helm chart for Heat, add the following information to the "osh" section under "charts":
+
+.. code-block:: yaml
+
+     heat:
+       location: https://git.openstack.org/openstack/openstack-helm
+       reference: ${COMMIT_ID}
+       subpath: heat
+       type: git
+
+.. note::
+
+   When specifying a particular version of a Helm chart, it may be necessary to first create the appropriate subsection under "charts". Airship components such as Deckhand and Shipyard belong under "ucp", OpenStack services belong under "osh", and infrastructure components such as MariaDB and RabbitMQ belong under "osh_infra".
 
```

Comment:
should we have an update airship ucp service section or combine into one?

Reply:
I was originally thinking separate sections, but since the process is the same for each, I decided to combine them into a general update section. I can definitely separate them if that makes more sense.